MOBILE_API.xml

```diff
@@ -3683,6 +3688,10 @@
     </struct>
 
     <enum name="LightName" since="5.0">
+    	<description>
+    		Enumeration that describes possible values of light name.
+    		Mobile libraries and core communicate using the name string as a value.
+    	</description>
         <!-- Common Single Light 0~500 -->
         <element name="FRONT_LEFT_HIGH_BEAM" value="0"/>
         <element name="FRONT_RIGHT_HIGH_BEAM" value="1"/>
```

Comment:
```suggestion
    <enum name="LightName" since="5.0">
        <description>
            Enumeration that describes possible values of light name.
            The mobile libraries and SDL Core use the name string when referencing these elements.
        </description>
        <!-- Common Single Light 0~500 -->
        <element name="FRONT_LEFT_HIGH_BEAM" value="0"/>
        <element name="FRONT_RIGHT_HIGH_BEAM" value="1"/>
```